mantidproject/mantid
# Update documentation following changes to ReflectometryISISCalibration

**Description of work**

This PR is branched from the changes in PR #35494. Only documentation updates need reviewing and **THIS SHOULD ONLY BE MERGED AFTER PR #35494**.

**Purpose of work**


Documentation updates required to support changes to `ReflectometryISISCalibration` in PR #35494.



**Summary of work**


- I've added a page for `ReflectometryISISCalibration` to provide detail on how it works and the supported calibration file format. A test calibration file is required for the usage example, so I followed the same approach as was used [here](https://docs.mantidproject.org/nightly/algorithms/UpdateInstrumentFromFile-v1.html) and included code in the example that creates the file.
- I've updated the `ReflectometryISISLoadAndProcess` docs to link to this page, as well as the release notes for PR #35494.
- I removed an incorrect statement from the docs for algorithm `ApplyCalibration`.


**To test:**


Read through the changes. To check how the pages are rendering, build the docs-html target.

Refs #35494. 






---

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
